src/views/Collectibles/components/NftList.tsx

```diff
@@ -13,21 +14,30 @@ import EasterNftCard from './NftCard/EasterNftCard'
  *
  */
 const nftComponents = {
-  10: BunnySpeciaCard,
-  11: BunnySpeciaCard,
-  12: EasterNftCard,
-  13: EasterNftCard,
-  14: EasterNftCard,
+  hiccup: BunnySpeciaCard,
+  bullish: BunnySpeciaCard,
+  'easter-storm': EasterNftCard,
+  'easter-flipper': EasterNftCard,
+  'easter-caker': EasterNftCard,
 }
 
 const NftList = () => {
-  const { nfts: nftTokenIds, refresh, lastUpdated } = useGetWalletNfts()
+  const { refresh, lastUpdated, getTokenIdsByIdentifier } = useGetWalletNfts()
 
   return (
     <NftGrid>
       {orderBy(nfts, 'sortOrder').map((nft) => {
-        const tokenIds = nftTokenIds[nft.bunnyId] ? nftTokenIds[nft.bunnyId].tokenIds : []
-        const Card = nftComponents[nft.bunnyId] || NftCard
+        const tokenIds = getTokenIdsByIdentifier(nft.identifier)
+
+        const getNftCardComponent = (sortedNft: Nft): FunctionComponent<NftCardProps> => {
+          if (!nftComponents[sortedNft.identifier]) {
+            return NftCard
+          }
+
+          return nftComponents[sortedNft.identifier]
+        }
```

Comment:
Perhaps this could make it clearer?

```suggestion
        const getNftCardComponent = (sortedNft: Nft): FunctionComponent<NftCardProps> => (
          nftComponents[sortedNft.identifier] ?? NftCard
        )
```

Reply:
You're right. The original code had more than one case I was checking and I was too lazy to move it back.